checkov/terraform/checks/resource/aws/RedshiftServerlessNamespaceKMSKey.py

```diff
@@ -0,0 +1,21 @@
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+from checkov.common.models.consts import ANY_VALUE
+
+
+class RedshiftServerlessNamespaceKMSKey(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name = "Ensure that Redshift serverless namespace is encrypted by KMS"
```

Comment:
```suggestion
        name = "Ensure Redshift Serverless namespace is encrypted by KMS using a customer managed Key (CMK)"
```
fits better to the other checks